plugins/outdated.yaml

```diff
@@ -43,6 +43,14 @@ spec:
     bin: outdated.exe
   shortDescription: Finds outdated container images running in a cluster
   homepage: https://github.com/replicatedhq/outdated
+  caveats: |
+    Usage:
+      $ kubectl outdated
+
+    For additional options:
+      $ kubectl outdated --help
```

Comment:
`caveats` is intended for must-read information after install. It is the right place for prerequisites or further installation steps.
So please move this back to the `description` where it fits better IMHO.